src/main/java/com/example/demo/exception/UserException.java

```diff
@@ -0,0 +1,38 @@
+package com.example.demo.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class UserException extends RuntimeException{
+
+	String message;
+	HttpStatus httpStatus;
+	
+	public UserException(String message, HttpStatus httpStatus) {
+		super();
+		this.message = message;
+		this.httpStatus = httpStatus;
+	}
+
+	public String getMessage() {
+		return message;
+	}
+
+	public void setMessage(String message) {
+		this.message = message;
+	}
```

Comment:
Remove setters. There is no use of them.

Reply:
Get  method we are using in Controller Advice to get the status

Reply:
setters mam 😂😂 padh toh le

Reply:
Okay okay😅